src/main/java/net/datafaker/providers/base/Internet.java

```diff
@@ -246,41 +246,29 @@ public String macAddress() {
      * @return a correctly formatted IPv4 address.
      */
     public String ipV4Address() {
-        try {
-            return getIpV4Address().getHostAddress();
-        } catch (UnknownHostException e) {
-            return "127.0.0.1";
-        }
+        return getIpV4Address().getHostAddress();
```

Comment:
I don't think this is ideal, I believe the code can throw exceptions now sometimes, and I don't think we should.

Reply:
I also had a doubt, but no, this code doesn't fail.

This is a test that tries through all possible values:
```java
    @Test
    void allPossibleAddresses() {
        for (int first = 2; first < 256; first++) {
            for (int second = 2; second < 256; second++) {
                for (int third = 2; third < 256; third++) {
                    for (int fourth = 2; fourth < 256; fourth++) {
                        InetAddress inetAddress = inet4Address((byte) first, (byte) second, (byte) third, (byte) fourth);
                        assertThat(inetAddress).isNotNull();
                    }
                }
            }
        }
    }
```